tests/unit/controllers/test_collection_instrument_controller.py

```diff
@@ -103,16 +103,6 @@ def test_upload_collection_instrument_success(self, *_):
                 except (ApiError, CiUploadError):
                     self.fail("Unexpected error thrown when uploading collection instrument")
 
-    @patch("frontstage.controllers.case_controller.post_case_event")
```

Comment:
Why is this being removed?

Reply:
Yeah test coverage could be improved with the new logic added above